app/uk/gov/hmrc/cgtpropertydisposals/models/Name.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cgtpropertydisposals.models
+
+import play.api.libs.json.{Json, OFormat}
+
+final case class Name (firstName : String, lastName : String)
```

Comment:
does scalafmt not take care of spaces between parameter names and colons?

Reply:
Yes it does. I've added `spaces.beforeContextBoundColon = Never` if you don't want a space after the identifier.

Reply:
it'd look better for consistency if we had something which took care of it. Everywhere else we have this sort of thing:
```scala
case class(a: A, b: B)
```
rather than 
```scala
case class(a : A, b: B)
```
with the spaces. It's still got the spaces in it

Reply:
That option does work for me. Let me know if it doesn't for you. If you use Intellij make sure you're IDE setting is set to you scalafmt.